assets/sass/main.min.scss

```diff
@@ -1,6 +1,7 @@
 @import '_config.scss';
 @import '_mixins.scss';
 @import '_responsive.scss';
+@import '_responsive-component-table.scss';
```

Comment:
I just divided included CSS files into multiple ones relevant to the current view. Per my comment above, stick it in pages/_docs.scss and then include that in the new file `assets/sass/docs.min.scss`.